shared/test/dataDriver_test.js

```diff
@@ -473,9 +503,22 @@ describe("loop.DataDriver", () => {
           userId: "theUserId"
         }));
     });
+
+    it("should throw an exception if value isn't valid", () => {
+      driver._processRecord("participant!theUserId", {
+        timestamp: 1234567890123,
+        value: 1234
```

Comment:
Technically the value can be of any value. E.g., if we had a meta!roomName record, the value would probably be a string and not an object.